### PR TITLE
Updated documentation for development

### PR DESCRIPTION
**Pull request by dlabordus**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+and [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) extensions in order to
+automatically lint and format your code as you edit it. There are similar plugins available for
+using [ESLint](https://eslint.org/) and [Prettier](https://prettier.io/) from within other IDEs and text editors.
+
+## Scripts
+
+We provide the following `npm` scripts for your convenience:
+
+- `npm start` runs `CoMPAS OpenSCD` for development, reloading on file changes
+- `npm test` runs the test suite with Karma
```
Comment (JakobVogelsang) [inline DEVELOPMENT.md:40]: Karma is replaced with web test runner

**Review by JakobVogelsang — APPROVED** [review]
In general LGTM. Two little comments from my side

File: DEVELOPMENT.md (hunk 1 above)
Comment (dlabordus) [reply DEVELOPMENT.md:40]: Fixed

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- `npm run lint` runs the linter (fixes problems in your code)
+- `npm run format` runs the formatter (formats your code in a unified way)
+- `npm run doc` builds HTML documentation into the `doc` directory
+- `npm run build` builds a deployable version of the project into the `dist` directory
```
[the remark this answers is not in the document]
Comment (dlabordus) [reply DEVELOPMENT.md:44]: Fixed

**Pull request merged**